src/ophys_etl/modules/suite2p_registration/suite2p_utils.py

```diff
@@ -272,9 +276,16 @@ def optimize_motion_parameters(initial_frames: np.ndarray,
                     'smooth_sigma': -1,
                     'smooth_sigma_time': -1}
     logger('Starting search for best smoothing parameters...')
+    sub_frames = load_representitive_sub_frames(
```

Comment:
spelling error "representative" ("at" not "it"). Sorry if that spirals out of control